### PR TITLE
PAYER-434: Updated betaBanner to be inline with GDS

### PR DESCRIPTION
**Pull request by johno1985**
Before:
before
After:
image

### CODE REVIEW

File: src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
 
-<div class="beta-banner@if(noBorder) { beta-banner--borderless}">
-    <p><strong class="phase-tag">BETA</strong><span>This is a new service.
-        @if( showFeedbackLinkInBetaBanner ) {
-            Help us improve it - send your
-            @defining(if (userLoggedIn) betaFeedbackUrl else betaFeedbackUnauthenticatedUrl) { linkUrl =>
+<div class="beta-banner @if(noBorder) {beta-banner--borderless}">
+    <p>
+        <strong class="phase-tag">BETA</strong>
+        <span>This is a new service @if(showFeedbackLinkInBetaBanner) {
```
Comment (feedmypixel) [inline src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html:25]: is this `<span>` needed?

**Review by feedmypixel — COMMENTED** [review]
Please attach a before and after screenshot to the description

File: src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html (hunk 1 above)
Comment (johno1985) [reply src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html:25]: I'm not the one to ask, that's what the existing implementation looks like and that's what http://govuk-elements.herokuapp.com/alpha-beta-banners/#beta-banner looks like

Comment (feedmypixel) [reply src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html:25]: The `<span>` has associated styles in assets-frontend https://github.com/hmrc/assets-frontend/blob/0551d318fe24a4ddb1fca4f45ff64a6ece265b11/assets/govuk_elements/govuk/public/sass/design-patterns/_alpha-beta.scss#L29-L38

Comment (johno1985) [reply src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html:25]: I'm not sure what you're alluding to here

Comment (feedmypixel) [reply src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html:25]: I'm checking that the `<span>` is needed and confirming it is needed.

**Comment by feedmypixel** [comment]
@johno1985 where has this change request come from? has this been signed off by @hmrc/designers ?

**Comment by johno1985** [comment]
I've already mentioned that on the team-bta channel

**Comment by feedmypixel** [comment]
> I've already mentioned that on the team-bta channel

Once confirmed you have sign off we can get this merged. Thanks.

**Comment by pietrodesiato** [comment]
The request is coming from @johno1985 's team. 
It looks like a simple content change to the banner with no changes to the layout, so fine by me.

**Comment by feedmypixel** [comment]
@pietrodesiato many thanks

**Pull request merged**